validator/validator-main.protoascii

```diff
@@ -2715,6 +2715,16 @@ attr_lists: {
   attrs: { name: "itemref" }
   attrs: { name: "itemscope" }
   attrs: { name: "itemtype" }
+  # Also for schema.org, RDFa syntax
+  attrs: { name: "about" }
+  attrs: { name: "content" }
```

Comment:
(Also, sorry for the duplicated git commit message, I did run `git commit --amend` on that one but clearly I failed at it.)